docs/source/reference/comparison.rst

```diff
@@ -1,15 +1,7 @@
 .. _sky-compare:
```

Comment:
nit: maybe also rename the rst file to have a better URL than comparison.html:
<img width="523" alt="image" src="https://github.com/user-attachments/assets/9789d376-b910-4565-aef0-8b66b5331da6" />

Reply:
Done.